src/status_im/ui/screens/wallet/navigation.cljs

```diff
@@ -16,8 +16,10 @@
   db)
 
 (defmethod navigation/preload-data! :wallet-request-transaction
```

Comment:
Can you explain this change? Why navigate back db and why different dissoc/wallet form?

Reply:
Always dissoc was a bug preventing to select an asset in `request` screen. 
The key change reflect a update to the app-db to align with new semantic (and reflect what is done in `send` screen).